doc/input-output-reference/src/overview/group-surface-construction-elements.tex

```diff
@@ -4115,7 +4115,7 @@ \subsubsection{Inputs}\label{inputs-38}
 
 \paragraph{Field: Tube Spacing}\label{field-tube-spacing}
 
-This field defines the distance between adjacent hydronic tubes spaced in the direction perpendicular to the main direction of heat transfer. The value for this parameter must be greater than 0.01m (or a tube spacing of 1 cm) and less than 1.0m (or a tube spacing of 1m). Note that this parameter is only used for two-dimensional solutions (see previous field).
+This field defines the distance between adjacent hydronic tubes spaced in the direction perpendicular to the main direction of heat transfer. The value for this parameter must be greater than 0.01m (or a tube spacing of 1 cm) and less than 1.0m (or a tube spacing of 1m). Note that this parameter is only used for two-dimensional solutions (see previous field) or when the user requests that the tube length for a hydronic radiant system ( \textbf{\hyperref[zonehvaclowtemperatureradiantvariableflow]{variable flow}} or  \textbf{\hyperref[zonehvaclowtemperatureradiantconstantflow]{constant flow}}) be autosized.  In the case of autosizing the tube length, this parameter is used along with the dimensions of the surface to approximate the tube length.
 
```

Comment:
If the model resets tube spacing to 0.075 when users enter too small of a value should that be stated here as well as with the new warning? I guess I can live with this since this text says the input "must be greater than 0.01m", although 10 cm would not trip a warning and is also valid. Details, details. Your doing a follow up on this anyway to change the idd and can revisit this if you think necessary.